docs/theme/src/main/twirl/lagom/documentation.scala.html

```diff
@@ -1,120 +1,252 @@
+                                    <a href="//stackoverflow.com/questions/tagged/lagom"><span>Stackoverflow</span></a>
+                                </nav>
+                                <nav class="sections-scroll">
+                                    
 
-                <aside class="left-off-canvas-menu">
-                    <ul class="off-canvas-list">
-                        <li><label>API docs</label></li>
-                        @for((href, text) <- apiDocs) {
-                            <li><a href="@href">@text</a></li>
-                        }
-                        @sidebar
-                    </ul>
-                </aside>
-                <div class="row">
-                    <div class="small-12 large-10 large-centered columns">
-
-                        <article id="page-content">
-                            @content
-
-                            @sourceUrl.map { url =>
-                                <hr />
-                                <p>Found an error in this documentation? The source code for this page can be found <a href="@url">here</a>. Please feel free to edit and contribute a pull request.</p>
-                            }
-                        </article>
+<a href="http://www.lagomframework.com/download.html">Download</a>
+<a href="/">Documentation</a>
+<a href="http://www.lagomframework.com/get-involved.html">Get Involved</a>
+<a href="http://www.lagomframework.com/blog/">Blog</a>
 
+                                </nav>
+                            </div>
+                        </div>
                     </div>
-                </div>
-                <footer class="site-footer">
-                    <div class="row">
-                        <div class="small-12 large-10 large-centered columns">
+
+                      
+    <header class="fw-wrapper lgm-purple page-title">
+        <div class="row">
+            <div class="small-12 columns">
+                <h1>@title.getOrElse("Documentation")</h1>
+            </div>
+        </div>
+    </header>
+
+    <div class="fw-wrapper documentation-content">
+        <div class="row">
+            <div class="small-12 large-7 columns">
+
+                @breadcrumbs
+
+                <article>
+                    @content
+                </article>
+
+                @for(url <- sourceUrl) {
+                    <p class="edit-source">Found an error in this documentation? The source code for this page can be found
+                        <a href="@url">here</a>. Please feel free to edit and contribute a pull request.</p>
+                }
+
+            </div>
+            <aside class="small-12 large-4 columns">
+                <nav class="side-menu">
+
+                    <input type="text" id="docs-search" placeholder="Search"/>
+
+                    <a href="/documentation/1.1.x/java/api/index.html" class="api-docs">API Documentation</a>
```

Comment:
Should this be serving up the local API docs? It just redirects back to the root page at the moment.